google-cloud-deploy/src/test/java/com/google/cloud/deploy/v1/MockCloudDeployImpl.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 Google LLC
+ * Copyright 2022 Google LLC
```

Comment:
Don't update the copyright year

Reply:
Why?

Reply:
This goes against our standards. We want the old year possible.